db/migrate/20160127193752_create_games.rb

```diff
@@ -0,0 +1,16 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.datetime :start_time, null: false
+      t.belongs_to :week, class_name: 'Week', null: false
```

Comment:
The difference here is a migration vs a model. That `class_name` code will work fine in a model, but won't do anything in a migration.